include/topic_tools/utilities.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 Daisuke Nishimatsu
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TOPIC_TOOLS__UTILITIES_HPP_
```

Comment:
does this need to be a public API? I would be inclined to hide it in `src` rather than exposing it publicly to consuming packages. That way we don't have to worry about breaking API/ABI if we change things.

Reply:
@emersonknapp I moved them into base node class.